device-types/FireBrick/FB2900.yaml

```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: FireBrick
+model: FB2900
+slug: fb2900
+part_number: FB2900
+u_height: 1
+is_full_depth: false
```

Comment:
Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not